apps/cms-v2/src/modules/providers/builder.io/builder-io.client.ts

```diff
@@ -79,10 +85,12 @@ export class BuilderIoClient {
   }
 
   async upsertProductVariant(variant: WebhookProductVariantFragment) {
+    this.logger.debug("upsertProductVariant called");
+
     const entriesToUpdate = await this.fetchBuilderIoEntryIds(variant.id);
 
     if (entriesToUpdate.length === 0) {
-      this.logger.debug("Didn't find any entries to update, will upload new variant");
+      this.logger.info("Didn't find any entries to update, will upload new variant");
```

Comment:
question: WDYT about info here, IMO it could be helpful to know what happend

Reply:
whatever makes observing easier